hbase-client/src/main/java/org/apache/hadoop/hbase/shaded/protobuf/ProtobufUtil.java

```diff
@@ -2345,14 +2345,6 @@ public static TableName toTableName(HBaseProtos.TableName tableNamePB) {
         .setQualifier(UnsafeByteOperations.unsafeWrap(tableName.getQualifier())).build();
   }
 
-  public static HBaseProtos.RegionInfo toProtoRegionInfo(
```

Comment:
Not used anymore? This seems not releated to HRegionInfo.

Reply:
We have two toRegionInfo methods for converting from RegionInfo to proto and vice versa. This method is not used and the implementation is wrong, so just remove it.